### ISSUE TITLE
Change rofication order to created_at desc

### ISSUE THREAD

**Issue by otavioschwanck** [regolith-linux/regolith-desktop#607]
make more sense to have the last notification on the top!

**Comment by gosirys** [comment]
yes i was looking into this to see if there was a way to change it in its configuration perhaps - if not i'll look for an alternative

**Comment by kgilmer** [comment]
@farsil what are your thoughts here?

**Comment by farsil** [comment]
I am not on regolith atm, but shouldn't the notifications be displayed in the order they are sent by the applications already?

**Comment by otavioschwanck** [comment]
> I am not on regolith atm, but shouldn't the notifications be displayed in the order they are sent by the applications already?

When i have more than 10+ notifications, i want to see from the latest to the oldest.  I jus want to see the newer ones, example:

If i receive a notification from slack, i open the rofitication, i have to go all the way down to use the action of notification (that opens the conversation).

Would be more simplier to just open the rofication and press enter to focus the conversation

**Comment by farsil** [comment]
Makes sense. I guess we need to reverse the list of notifications before displaying them.

**Comment by kgilmer** [comment]
Well, ideally we could expose this as an Xresource setting so that users could override the default if they prefer a different sort order.  I'm sorry my Python skills are all but forgotten.  I looked over the various files @farsil but wasn't sure of the best place to enable a sort of the queue before being sent to the "gui client".  If you could point me to where in the code you'd suggest and a rough idea of what functions I should use I can probably take care of the rest @farsil .  Or, if you'd like to do it you're more than welcome :)  TIA

**Comment by farsil** [comment]
I would gladly do it myself, the problem is that right now I'm relocating and I have little time for side projects :( If you don't mind waiting a couple of weeks I can do it, otherwise you can add the resource in `rofication/resources/_static.py` and reverse the list at line 42 in `_server.py` if the resource is set.

**Comment by dferrand** [comment]
I created a the following PR https://github.com/regolith-linux/regolith-rofication/pull/18 to address this issue.

**Comment by cfsmp3** [comment]
> I created a the following PR [regolith-linux/regolith-rofication#18](https://github.com/regolith-linux/regolith-rofication/pull/18) to address this issue.

Would you consider adding sorting based on fields? Your PR works but it sorts by order or arrival (ascending or descending). I think it would be useful to expand this to support urgency, timestamp (or better, a combination of both).

**Comment by dferrand** [comment]
It is now possible to sort on multiple fields. The default sort is by timestamp descending (newest messages first).

**Comment by kgilmer** [comment]
UI has changed for Regolith in version 2.  This feature is not implemented but will be tracked for the future.
